SPECS/xcp-ng-release.spec

```
@@ -723,6 +724,9 @@ systemctl preset-all --preset-mode=enable-only || :
 
 # Keep this changelog through future updates
 %changelog
+* Mon Jan 24 2022 Gael Duperrey <gduperrey@vates.fr> - 8.2.1-2
```

```suggestion
* next
```

We'll update the changelog line the next time we build it